test/index.test.ts

```diff
@@ -21,149 +23,187 @@ const testCases = [
 
 const eip191Signer = new EIP191Signer();
 
-describe('Hash message function', () => {
+describe('function hashEthereumSignedMessage', () => {
   it('should be hexadecimal and of 32 bites', () => {
     testCases.forEach((data) => {
-      const hash = eip191Signer.hashEthereumSignedMessage(data);
+      const hashedData = eip191Signer.hashEthereumSignedMessage(data);
 
-      expect(hash.substring(0, 2)).toBe('0x');
-      expect(hash.length).toBe(66);
+      expect(hashedData.substring(0, 2)).toBe('0x');
+      expect(hashedData.length).toBe(66);
     });
   });
 
-  it('should be prefixed with "\x19Execute Relay Call:\n"', () => {
+  it('should be prefixed with "\x19\x45thereum Signed Message:\n"', () => {
     const hashedMessage = eip191Signer.hashEthereumSignedMessage('Hello World');
 
     expect(hashedMessage).toBe(
-      '0x677739c1b99336b0c40ed12a4d77c68805f8b5ca2d865676de85bf83b3b664ee',
+      '0xa1de988600a42c4b4ab089b619297c17d53cffae5d5120d82d8a92d0bb3b78f2',
     );
   });
 });
 
-describe('Sign ethereum signed message function', () => {
-  testCases.forEach((data) => {
-    const signedObject = eip191Signer.signEthereumSignedMessage(data, signingKey);
+describe('function hashDataWithIntendedValidator', () => {
+  it('should be hexadecimal and of 32 bites', () => {
+    testCases.forEach((data) => {
+      const hashedData = eip191Signer.hashDataWithIntendedValidator(
+        validatorAddress,
+        data,
+      );
 
-    it('should return an object with the expected properties', () => {
-      expect(signedObject).toHaveProperty('message');
-      expect(signedObject).toHaveProperty('messageHash');
-      expect(signedObject).toHaveProperty('v');
-      expect(signedObject).toHaveProperty('r');
-      expect(signedObject).toHaveProperty('s');
-      expect(signedObject).toHaveProperty('signature');
+      expect(hashedData.substring(0, 2)).toBe('0x');
```

Comment:
maybe a better test is to use: https://web3js.readthedocs.io/en/v1.2.11/web3-utils.html#ishexstrict

Reply:
so you basically check for a hex rather than just testing for 0x prefix